tests/auth/test_generic_auth.py

```diff
@@ -187,8 +187,8 @@ def test_update_account_when_no_server_provided(db):
     assert acc_smtp_port == smtp_port
 
 
-@pytest.mark.networkrequired
-def test_double_auth(db):
+@pytest.mark.usefixtures('mock_smtp_get_connection')
```

Comment:
FYI, you can just pass the fixture to the function instead of adding a decorator.